build failure (dependency URL change)

```[ERROR] Failed to execute goal on project verapdf-rest: Could not resolve dependencies for project org.verapdf:verapdf-rest:jar:0.1.0-SNAPSHOT: Failed to collect dependencies at com.yunspace.dropwizard:dropwizard-xml:jar:41: Failed to read artifact descriptor for com.yunspace.dropwizard:dropwizard-xml:jar:41: Could not transfer artifact com.yunspace.dropwizard:dropwizard-xml:pom:41 from/to jcenter (http://jcenter.bintray.com): Access denied to: http://jcenter.bintray.com/com/yunspace/dropwizard/dropwizard-xml/41/dropwizard-xml-41.pom -> [Help 1]```

https://jfrog.com/blog/secure-jcenter-with-https/

--- a/pom.xml
+++ b/pom.xml
@@ -42,7 +42,7 @@
     </repository>
     <repository>
       <id>jcenter</id>
-      <url>http://jcenter.bintray.com</url>
+      <url>https://jcenter.bintray.com</url>
     </repository>
   </repositories>
 

